app/validators/questionnaire_validator.py

```diff
@@ -107,6 +108,47 @@ def validate_smart_quotes(self):
                         pointer=translatable_item.pointer,
                     )
 
+    def validate_html(self):
+        html_strings = []
+        schema_object = SurveySchema(self.schema_element)
+
+        # pylint: disable=invalid-string-quote
+        html_regex = re.compile(r"<[^>]*>")
+
+        for translatable_item in schema_object.translatable_items:
+            schema_text = translatable_item.value
+
+            values_to_check = [schema_text]
+
+            if isinstance(schema_text, dict):
+                values_to_check = schema_text.values()
+
+            html_strings.extend(
+                {"pointer": translatable_item.pointer, "text": schema_text}
+                for schema_text in values_to_check
+                if schema_text and html_regex.search(schema_text)
+            )
+        if html_strings:
+            self.check_invalid_html_tags(html_strings)
+
+    def check_invalid_html_tags(self, html_strings):
+        strong = re.compile(r"<strong>(?:(?!</strong>).)*</strong>")
+        anchor = re.compile(r"<a [^>]*>.*?</a>")
```

Comment:
Since anchor [always takes `href` argument only](https://www.w3schools.com/tags/att_a_href.asp) can we add the href as something always present in this regex?:
```suggestion
        anchor = re.compile(r"<a href[^>]*>.*?</a>")
```

Reply:
Reverted as it leads to some false positives where href is present but not the first argument